### PR TITLE
Changed theme to RTD and created quick start guide

### PR DESCRIPTION
**Pull request by lilyminium**
From #1 :

> **Story:** *As a* molecular dynamics user, *I want* to set up MDAnalysis quickly and easily, *so that* I can move onto further analysis.

>  **Story:** *As a* molecular dynamics user, *I want* to convert my AMBER trajectory into a GROMACS format, *so that* I can use the GROMACS analysis tools on it. 

**Original plan**
> **Sections:**
> 
> 1. Link to page on installation.
> 2. Load in a Universe.
> 3. Writing out coordinates.
> 4. Basic introduction to AtomGroups, ResidueGroups, SegmentGroups.
> 5. Selecting atoms.
> 6. Trajectories and positions.
> 7. Universe persistence.
> 8. Introduction to Analysis and modules.

**Current structure:**
* Overview
* Loading a structure or trajectory
* Working with groups of atoms
    * Selecting atoms
    * Getting atom information from AtomGroups
    * AtomGroup positions and methods
* Working with trajectories
    * Dynamic selection
* Writing out coordinates
    * Single frame
    * Trajectories
* Analysis
    * RMSD
* References
    * Automatic citations with duecredit

While the general gist of the original plan is here, the actual structure is somewhat out the window. I struggled to place coordinate output early on without interrupting the flow. For users that see MDAnalysis as a way to convert trajectories, a nifty command-line tool might be better. Otherwise they can use the side bar to click directly to Writing out coordinates.

**Acceptance criteria:**

- [x] I know how to load in a coordinate file / simulation and write it out
    - [ ] almost immediately
- [ ] I understand the arguments necessary to create ('set up') a Universe (kind of)
- [x] I understand basic atom selection
- [x] I understand basic analysis
- [x] I know how to write out coordinates.
- [x] I know where to look for less general knowledge (e.g. complex selection).
- [x] I have links to an index of examples, available analyses, ~and a tutorial for developing custom analyses.~

### CODE REVIEW

**Comment by lilyminium** [comment]
**Theme**

I changed the theme from alabaster to readthedocs, modifying the CSS for sufficient orange. `sphinx_rtd_theme` is an additional extension. I also added `nbsphinx` earlier for pages that are also notebooks (the quick start guide, for example, is a notebook that users could download and work through.) Here's a screenshot of how it looks to me:

Screenshot 2019-09-09 at 10 53 34 PM

The orange links would benefit from being darker on that grey background, and suddenly the code highlighting clashes, but overall I find the collapsible menu with high contrast text/background colours to be more readable. Any thoughts?

**Comment by orbeckst** [comment]
@beckstein what was the font that you used for the original MDAnalysis label?

**Comment by orbeckst** [comment]
The new theme looks good. I do like the menu block. It's overall appearance is quite bold – I like it.

I also agree that collapsible menus are a massive improvement in navigability (if this is a word).

File: doc/examples/analysis/.ipynb_checkpoints/pca-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,344 @@
+{
```
Comment (orbeckst) [inline doc/examples/analysis/.ipynb_checkpoints/pca-checkpoint.ipynb]: delete checkpoint files (exclude in `.gitignore`)

File: doc/examples/analysis/.ipynb_checkpoints/PCA_Tutorial-checkpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,689 @@
+{
```
Comment (orbeckst) [inline doc/examples/analysis/.ipynb_checkpoints/PCA_Tutorial-checkpoint.ipynb]: delete checkpoint files (exclude in `.gitignore`)

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (orbeckst) [inline .vscode/settings.json]: delete build specific files (exclude in `.gitignore`)

File: doc/examples/analysis/pca.ipynb [hunk 4]
```diff
@@ -0,0 +1,344 @@
+{
```
Comment (orbeckst) [inline doc/examples/analysis/pca.ipynb]: delete checkpoint files (exclude in `.gitignore`)

File: doc/examples/analysis/PCA_Tutorial.ipynb [hunk 5]
```diff
@@ -0,0 +1,689 @@
+{
```
Comment (orbeckst) [inline doc/examples/analysis/PCA_Tutorial.ipynb]: delete checkpoint files (exclude in `.gitignore`)

File: doc/source/_static/custom.css [hunk 6]
```diff
@@ -5,8 +5,9 @@
 /* MDAnalysis gray: #808080 */
 /* MDAnalysis white: #FFFFFF */
 /* MDAnalysis black: #000000 */
+/* Darker orange: e76900 */
 
-@import url('readable.css');
```
Comment (orbeckst) [inline doc/source/_static/custom.css]: I would completely delete, not just comment out.

File: doc/source/auto_examples/index.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
+:orphan:
+
```
Comment (orbeckst) [inline doc/source/auto_examples/index.rst]: Remove this file? (And various zip files)?

**Comment by orbeckst** [comment]
@lilyminium thanks for your patience, waiting for feedback. However, it's totally fine to ping me again if I haven't done anything for two days....

**Comment by lilyminium** [comment]
> I like the new theme.
> 
> I would suggest to remove a whole bunch of files from the commit (vscode, notebook checkpoints, example files, zip files, ...).

Oops, that's what I get for early-morning commits. The files in auto_examples are generated by `sphinx_gallery` to create the 'Examples' page of notebooks. The zips are a collection of all of them -- this a configurable option so we can turn it off.

Screenshot 2019-09-22 at 10 44 46 AM

**Comment by lilyminium** [comment]
> @lilyminium thanks for your patience, waiting for feedback. However, it's totally fine to ping me again if I haven't done anything for two days....

@orbeckst Yes, similar (sorry!) -- life has a way of getting in the way of my to-do list.

**Comment by beckstein** [comment]
@orbeckst:
please find attached the original font used as basis for the logo-illustration 
its named „ZAG Bold“

https://www.dafont.com/de/zag.font <https://www.dafont.com/de/zag.font>

Grtz Chris 



—————————————————————————

**Comment by orbeckst** [comment]
Thanks @beckstein !

**Comment by orbeckst** [comment]
@lilyminium the theme at https://www.mdanalysis.org/UserGuide/ looks really good. 

I will vote for changing all our docs to your theme. I am pretty sure you'll hear similar sentiments from others, too, once they can experience the [UserGuide](https://www.mdanalysis.org/UserGuide/) layout and style for themselves.

We should then also to apply the altered color scheme to the webpage itself.

**Pull request merged**